### ISSUE TITLE
Tick Bars

### ISSUE THREAD

**Issue by durana** [BlackArbsCEO/Adv_Fin_ML_Exercises#2]
Unless I'm misunderstanding something, I believe there's an error in the function that calculates the indices for tick bars.

I think the `ts += i` in the for-loop should be `ts += 1` so that you accumulate a count of ticks, and not a sum of index values.

```python
def  tick_bars(df, column, m):
    '''
    compute tick bars
    
    # args
        df: pd.DataFrame()
        column: name for price data
        m: int(), threshold value for ticks
    # returns
        idx: list of indices
    '''
    t = df[column]
    ts = 0
    idx = []
    for i, x in enumerate(tqdm(t)):
        ts += i
        if ts >= m:
            idx.append(i)
            ts = 0
            continue   
    return idx
```

**Comment by BlackArbsCEO** [comment]
@durana Yes you're correct. Thank you. Will update later today with imbalance bar implementation.

**Comment by BlackArbsCEO** [comment]
closed #ef219be
